README.md

```diff
@@ -50,3 +50,54 @@ Running slow (overnight) tests:
 ```
 make runslow | tee raport.txt
 ```
+
+
+# Deploying with truffle
+### Installation
+Requires node.js >= 8
+
+Install dependencies:
+```
+npm install
+```
+
+
+### Deploying
+Deploying the contracts requires three accounts:
+1. `DEPLOYER` The account that actually deploys the contracts
+2. `AUTHORITY` The Authority account calls `RootChain.init()` and is the account used by the Child chain (or operator). By default a new `AUTHORITY` account is created when deploying. If you prefer you can use an existing `AUTHORITY` account, but it must not have made any transaction prior to calling `RootChain.init()` i.e. its nonce must be 0.
+3. `FAUCET` If a new `AUTHORITY` account is created, the `FAUCET` account is used to fund it. This account can be the same as the `DEPLOYER` account.
```

Comment:
I can't imagine a case where we would need faucet to be different from deployer. Why not simplify?